bentoml/_internal/frameworks/sklearn.py

```diff
@@ -11,6 +11,7 @@
 from bentoml.exceptions import NotFound
 from bentoml.exceptions import BentoMLException
 from bentoml.exceptions import MissingDependencyException
+from bentoml._internal.models.model import ModelSignaturesType
```

Comment:
@sauyon  @larme let's use this type for `signature` parameter in all `save_model` methods?